examples/extensions/update_sitelink.rb

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env ruby
+# Encoding: utf-8
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Update the sitelink extension feed item with the specified link text.
+
+require 'optparse'
+require 'google/ads/google_ads'
+
+def update_sitelink(customer_id, feed_item_id, sitelink_text)
+  # GoogleAdsClient will read a config file from
+  # ENV['HOME']/google_ads_config.rb when called without parameters
+  client = Google::Ads::GoogleAds::GoogleAdsClient.new
+
+  operation = client.operation.update_resource.extension_feed_item(
+    client.path.extension_feed_item(customer_id, feed_item_id)
+  ) do |efi|
+    efi.sitelink_feed_item = client.resource.sitelink_feed_item do |sitelink|
+      sitelink.link_text = sitelink_text
```

Comment:
We might need a  nil check for sitelink_text here.
Or we can add a default value for options[:sitelink_text] in main.